### ISSUE TITLE
Tracking for API v10 migration

### ISSUE THREAD

**Issue by veni-vidi-code** [nextcord/nextcord#467]
This reflects the changes that need to / have already been done for v10. Current branch for this is https://github.com/nextcord/nextcord/tree/community/apiv10

For more info about changes see https://github.com/discord/discord-api-docs/discussions/4510

- [x] Bump version in urls
- [x] Message Intent (Will be done in #457)
- [x] `application.summary` is depreceated and should be removed (Will be done in #464 )
- [x] document removal of `application.summary`
- [x] ~~Achievement localization format has changed. name and description are now strings, and localized strings are now stored in name_localizations and description_localizations~~ Game SDK only
- [x] ~~`/channels/<channel_id>/threads/active` has been removed~~ we do not use that
- [x] Existing attachments must be specified when PATCHing messages with new attachments. Any attachments not specified will be removed and replaced with the specified list
- [x] Audit log reason as a body/query parameter is no longer supported, and you should instead use the X-Audit-Log-Reason header.  (Will be done in #466)
- [x] Message routes accept embeds rather than embed (Will be done in #464 )
- [x] Requests to v10 and higher will no longer be supported on discordapp.com 
- [ ] Test the code
- [x] Document in changelog
- [x] Wait until it is actually the recommended API Version

**Comment by MaskDuck** [comment]
hm, i should have removed document of `AppInfo.summary`.

**Comment by MaskDuck** [comment]
> Existing attachments must be specified when PATCHing messages with new attachments. Any attachments not specified will be removed and replaced with the specified list

i dont think it needs to have any code change, since patching message means editing message using `Message.edit` if i do remember correctly. so i think we only need to document this one.

**Comment by MaskDuck** [comment]
i think i will document it right now.

**Comment by DenverCoder1** [comment]
> Existing attachments must be specified when PATCHing messages with new attachments. Any attachments not specified will be removed and replaced with the specified list

This is done in #525

**Comment by DenverCoder1** [comment]
> application.summary is depreceated and should be removed

It seems we will not be removing it yet. Discord still returns it in v10 (will be removed in v11)

>  document removal of application.summary

Will be done in #552

> Message routes accept embeds rather than embed

Already done (See https://github.com/nextcord/nextcord/commit/47e6a754e496a811df0f29c49bf3fd9512d20021)

When support for multiple embeds was added, the embed field was no longer in use

**Comment by ooliver1** [comment]
>  Wait until it is actually the recommended API Version

should we still wait on this? its been a while and i dont think v10 is going to appear in docs that soon

**Comment by ooliver1** [comment]
https://github.com/discord/discord-api-docs/commit/9759009999883a70b4ba26d66e3d654920c309c3 Now testing is all that's left
